pkg/api/index.go

```diff
@@ -470,6 +470,7 @@ func (hs *HTTPServer) setIndexViewData(c *models.ReqContext) (*dtos.IndexViewDat
 		Sentry:                  &hs.Cfg.Sentry,
 		Nonce:                   c.RequestNonce,
 		ContentDeliveryURL:      hs.Cfg.GetContentDeliveryURL(hs.License.ContentDeliveryPrefix()),
+		LoadingLogo:             "public/img/grafana_icon.svg",
```

Comment:
Shouldn't this be read from the settings and the default file url instead be specified in defaults.ini? Otherwise the url set in custom.ini has no effect.

Reply:
I used settings here https://github.com/grafana/grafana-enterprise/pull/1654. I don't know if its that you looking for.